charts/community-operator/templates/mongodbcommunity_cr_with_tls.yaml

```diff
@@ -75,5 +75,11 @@ spec:
       {{- end }}
     authentication:
       modes: ["SCRAM"]
-  users: []
+  {{- with .Values.resource.users }}
+  users:
+  {{- toYaml . | nindent 4 }}
+  {{- end }}
+  
+  additionalMongodConfig:
```

Comment:
Removed the `additionalMongodConfig`. Was definitely accidental. The `users` already existed. I simply made it controllable from `values` files so that the end users don't have to fork the chart to add in a user.